classes/task/reminder.php

```diff
@@ -52,16 +52,17 @@ public function get_name() {
     public function execute() {
         global $DB;
         $kanbancards = $DB->get_records_sql(
-            'SELECT ' . $DB->sql_concat('c.id', '"-"', 'a.userid') . ' as uniqid,
+            'SELECT ' . $DB->sql_concat('c.id', '\'-\'', 'a.userid') . ' as uniqid,
```

Comment:
Everywhere else in the moodle core code similar issues are being solved by using
```suggestion
            'SELECT ' . $DB->sql_concat('c.id', "'-'", 'a.userid') . ' as uniqid,
```
So I suggest to use this solution.

BTW: I tried to do some research and it looks like (I could not really find a reliable source for this information) according to the ANSI standard escaping should be done by using double single quotes. Using a backslash only seems to be supported by MySQL and postgres, but, for example, not on Oracle (this information might be wrong or outdated). But let's go with the solution mentioned above anyway :-)

Reply:
When using '\'', escaping is done by PHP so there is no SQL issue - but nevertheless, I'm choosing the "Moodle standard" ;-)

Reply:
ah, you're right of course. But yes, moodle "standard" :-)